### PR TITLE
Add warning message in compatibility table.

### PR DESCRIPTION
**Pull request by lipingxue**
Signed-off-by: Liping Xue lipingx@vmware.com

**What this PR does / why we need it**:
We observed that  Velero vSphere Operator Pod will restart periodically when using Velero vSphere Operator 1.3.0 with vSphere version 7.0U3h. This change is to add a warning message in the compatibility table for supervisor cluster doc.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```NONE

```

### CODE REVIEW

File: docs/supervisor.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
...
 |-----------------------------------|---------------------|-------------------------------------------------------------------|----------------------------|----------------|---------------------------------|---------------|------------|---------------|
 | 1.4.2                             | 8.0                  | Bundled with vSphere (1.22-1.23)                                 | Bundled with vSphere       | 1.9.2          | 1.3.0                           | 1.2.0        | No         | N/A           |
-| 1.4.2                             | 7.0U3h               | Bundled with vSphere (1.22)                                      | Bundled with vSphere       | 1.9.2          | 1.3.0                           | 1.2.0        | No         | N/A           |
+| 1.4.2                             | 7.0U3h  (**Warning:** Velero vSphere Operator Pod will restart periodically )             | Bundled with vSphere (1.22)                                      | Bundled with vSphere       | 1.9.2          | 1.3.0                           | 1.2.0        | No         | N/A           |
```
Comment (xing-yang) [inline docs/supervisor.md:17]: Can you add that "Velero can only be installed while the Velero vSphere Operator Pod is running."

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/809/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/810/

**Review by xing-yang — APPROVED** [review]
LGTM

**Pull request merged**